corehq/apps/cleanup/tasks.py

```diff
@@ -1,10 +1,47 @@
+import os
+import json
+from time import time
 from collections import defaultdict
 
+from celery.schedules import crontab
+from celery.task import periodic_task
+
+from django.core.management import call_command
+
+from corehq.apps.hqwebapp.tasks import mail_admins_async
 from corehq.apps.cleanup.management.commands.fix_xforms_with_undefined_xmlns import \
     parse_log_message, ERROR_SAVING, SET_XMLNS, MULTI_MATCH, \
     CANT_MATCH, FORM_HAS_UNDEFINED_XMLNS
 
 
+UNDEFINED_XMLNS_LOG_DIR = '/tmp/'
```

Comment:
one thing to note though is that it'll be hard to rotate those logs unless we do significant modifications to the original command. (we rely on our python log handlers to do rotation) so these files will be sitting around in the `log` dir until someone decides to clean them out